ci/build/build-docker-images.py

```diff
@@ -94,6 +94,10 @@ def _release_build():
     return branch and branch.startswith("releases/")
 
 
+def _pr_build():
+    return os.environ.get("BUILDKITE_PULL_REQUEST") != "false"
```

Comment:
`_is_pr_build()` ? but I guess there is already `_release_build()` so.. up to you.

maybe you can just make it a global variable for read, rather than trying to read the env var every time?